Update controlz interface img and output command (#7859)
istio/istio.io: content/en/docs/ops/diagnostic-tools/controlz/index.md

> of a running component. Components open a port which can be used from a web browser to get an interactive view into the state of the

Reviewer: Is istiod the only component that this applies to now? If so, should we change the title, description, and first paragraph to be more specific?
```
title: Istiod Introspection
description: Describes how to use ControlZ to get insight into a running istiod component.
...
Istiod is build with a flexible introspection framework, called ControlZ, which makes it easy to inspect and manipulate the internal state
of an istiod instance. Istiod opens a port which can be used from a web browser to get an interactive view into its state,
or via REST for access and control from external tools.

When Istiod starts, a message is logged indicating the IP address and port to connect to in order to interact with ControlZ.
```

Author: AFAIK it's true. Thanks.

component, or via REST for access and control from external tools.

> Istiod implement the ControlZ functionality. When the component start, a message is logged indicating the

Reviewer: ```suggestion
Istiod implement the ControlZ functionality. When it starts, a message is logged indicating the
```

IP address and port to connect to in order to interact with ControlZ.

{{< text plain >}}
2020-08-04T23:28:48.889370Z     info    ControlZ available at 100.76.122.230:9876
{{< /text >}}

Here's sample of the ControlZ interface:

{{< image width="90%" link="./ctrlz.png" caption="ControlZ User Interface" >}}

> To access the ControlZ page of deployed component (i.e. istiod), you can port-forward their ControlZ endpoints

Reviewer: ```suggestion
To access the ControlZ page of istiod, you can port-forward its ControlZ endpoint
```

locally and connect through your local browser:

{{< text bash >}}

> $ istioctl dashboard controlz istiod-954c974bb-qgb4h -n istio-system

Reviewer 2: Not sure why the <podname> was changed, although I know when automating a page, the <> in the bash has lint issues. In this cases, the <>'s have just been removed so this line would have become `istioctl dashboard controlz podium`

Author: This page was very old and haven't been updated for a long time. As all the functionality is combined into `istiod`, it's better to mention `podname` directly in the command. Also, the namespace flag was missing.

> $ istioctl dashboard controlz istiod-954c974bb-qgb4h -n istio-system

Reviewer: ```suggestion
$ istioctl dashboard controlz <istiod-podname> -n istio-system
```
@ericvn now that we support `snip_id=none`, we can use it for snippets with invalid bash syntax, e.g., `<podname>`, instead of removing the `<` and `>`. Since this kind of example is not a runnable command, as is, there's no need to generate the snip anyway.

Author: @frankbu Is it ok to name it like `istiod-<podname>` as `istiod` will not change?

Reviewer: @shamsher31 That's not quite right, because the `<...>` part would not be a podname, but rather a pod name suffix. Maybe a better (less confusing) syntax would be `<istiod pod name>`?

Reviewer: Updated suggestion above

Author: Thanks
